scripts/run_on_popular_crates/src/main.rs

```diff
@@ -0,0 +1,100 @@
+use std::time::Duration;
+use std::collections::HashMap;
+use std::process::Command;
+use std::io::Write;
+use crates_io_api::{CratesQuery, Sort};
+
+/// To get the rustdoc of the baseline, we first create a placeholder project somewhere
+/// with the baseline as a dependency, and run `cargo rustdoc` on it.
+fn create_rustdoc_manifest_for_crate_version(
+    name: &str,
+    version: &str,
+    features: &HashMap<String, Vec<String>>
+) -> cargo_toml::Manifest<()> {
+    use cargo_toml::*;
+
+    Manifest::<()> {
+        package: {
+            let mut package = Package::new(name, version);
+            package.publish = Inheritable::Set(Publish::Flag(false));
+            Some(package)
+        },
+        workspace: Some(Workspace::<()>::default()),
+        lib: {
+            let product = Product {
+                path: Some("lib.rs".to_string()),
+                ..Product::default()
+            };
+            Some(product)
+        },
+        dependencies: {
+            let project_with_features = DependencyDetail {
+                version: Some(version.to_string()),
+                features: features
+                    .iter()
+                    .map(|(key, _values)| key.clone())
+                    .collect(),
+                ..DependencyDetail::default()
+            };
+            let mut deps = DepsSet::new();
+            deps.insert(
+                name.to_string(),
+                Dependency::Detailed(project_with_features),
+            );
+            deps
+        },
+        ..Default::default()
+    }
+}
+
+fn save_manifest(manifest: cargo_toml::Manifest<()>, path: &str) -> anyhow::Result<()> {
+    std::fs::write(path, toml::to_string(&manifest)?)?;
+    Ok(())
+}
+
+fn run_checks_on_manifest() -> anyhow::Result<()> {
+    let output = Command::new("../../../target/debug/cargo-semver-checks")
```

Comment:
This seems to assume that a `cargo build` has already happened in debug mode, and would fail with an opaque error message if that isn't the case.

Perhaps it would be better to run a `cargo run` instead?

Reply:
Done in 81b00a95ec7520843f9cc283b042f4e246335713.